main/src/modules/contextualLayer/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright: (C) 2014 WYSIWYD Consortium
+# Authors: Low Sock Ching
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+
+#add_subdirectory(goalManager)
```

Comment:
Again, why is `goalManager` commented out? Is it not ready yet? If so, this PR should not contain any references to it.